src/badger-across/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+generated
+build
+subgraph.yaml
```

Comment:
If we don't have templates we need to include the subgraph.yaml file.  Will remove once we have templates.